src/Symfony/Bundle/WebProfilerBundle/composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": "^7.1.3",
         "symfony/config": "^4.2|^5.0",
-        "symfony/http-kernel": "^4.3|^5.0",
+        "symfony/http-kernel": "^4.3",
```

Comment:
because of https://github.com/symfony/symfony/blob/4.4/src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php#L63

Reply:
Would other projects be affected in the same way ? This would tell me that we miss a migration path here.

Reply:
The issue exists only if listener methods are extensibility point by inheritance. That's not the case most of the time, and even if it is, there are ways around.

Reply:
great that we marked our listeners as `@final` to avoid similar issues in the future...